README.md

```diff
@@ -104,19 +106,37 @@ Target SC2 version: 5.0.5
 However, sometimes one may need to compile with an older version of the game, e.g. to play with a Linux build which is
 always behind the Windows version. It is possible by specifying the game version manually, e.g.:
 ```bash
-$ cmake -DSC2_VERSION=4.10.0 ../
+$ cmake -DSC2_VERSION=4.10.0
```

Comment:
In this case, examples should use `-B build` as we are outside of the build folder.
Here and below.